src/event_form/components/image-upload/event-form-image-upload.html

```diff
@@ -15,30 +14,36 @@ <h4 class="modal-title" ng-bind="::modalTitle"></h4>
     </p>
     <div ng-hide="showAgreements">
       <div class="form-group">
-        <label for="inputFile">Selecteer je foto('s)</label>
-        <input type="file"
-               id="inputFile"
-               ng-model="imagesToUpload"
-               multiple
-               ng-file-select
-               ng-multiple="true"
-               accept="'image/*'"
-               ng-change="clearError()">
+        <label for="inputFile">Selecteer je foto</label>
+        <p>
+          <button type="file"
+                  class="btn btn-primary"
+                  id="inputFile"
+                  ngf-select="selectFile($file, $invalidFiles)"
+                  accept="image/*"
+                  ngf-max-size="1MB">
```

Comment:
Maken we dit best niet configureerbaar zoals op de back-end? De error-boodschap dan zou ook wel dynamisch moeten zijn.